index.html

```diff
@@ -3,9 +3,143 @@
+            </div>
+            <div class="table-row">
+                <div class="table-cell first-cell">
+                    <p>European Union Euro Currency</p>
+                </div>
+                <div class="table-cell">
+                    <p>1 EUR</p>
+                </div>
+                <div class="table-cell">
+                    <p class="euro">TL</p>
+                </div>
+            </div>
+            <div class="table-row">
+                <div class="table-cell first-cell">
+                    <p>Swedish Krona Currency</p>
+                </div>
+                <div class="table-cell">
+                    <p>1 SEK</p>
+                </div>
+                <div class="table-cell">
+                    <p class="krona">TL</p>
+                </div>
+            </div>
+            <div class="table-row">
+                <div class="table-cell first-cell">
+                    <p>China Yuan Currency</p>
+                </div>
+                <div class="table-cell">
+                    <p>1 CNY</p>
+                </div>
+                <div class="table-cell">
+                    <p class="yuan">TL</p>
+                </div>
+            </div>
+            <div class="table-row">
+                <div class="table-cell first-cell">
+                    <p>Japanese Yen Currency</p>
+                </div>
+                <div class="table-cell">
+                    <p>1 JPY</p>
+                </div>
+                <div class="table-cell">
+                    <p class="yen">TL</p>
+                </div>
+            </div>
+        </div>
+        <!-- converter section -->
+        <div class="converter-container">
+            <div class="wrapper">
+                <div class="converter-heading">
+                    <h1>Currency Converter</h1>
+                    <p>Please enter the currency you want to convert</p>
+                </div>
+                <form id="form">
+                    <div class="amount">
+                        <p>Enter Amount</p>
+                        <input type="text" value="1" class="amount-input">
+                    </div>
+                    <div class="drop-list-container">
+                        <div class="from-drop-list">
+                            <p>From</p>
+                            
+                                <select class="select-box" name="select" id="from-select"></select>
+                            
+                        </div>
+                        <div class="icon"><i class="fas fa-exchange-alt"></i></div>
+                        <div class="to-drop-list">
+                            <p>To</p>
+                         
+                            <select class="select-box" name="select" id="to-select"></select>
+                        </div>
+                    </div>
+                    <div class="exhange-rate-text">Getting exchange rate...</div>
+                    <button class="form-button">Swap</button>
```

Comment:
Also, add type submit to this button as the expectation is that the button triggers submitting the form.
```suggestion
                    <button type="submit" class="form-button">Swap</button>
```